### ISSUE TITLE
Incorrect status code for queries

### ISSUE THREAD

**Issue by jacksontj** [Comcast/trickster#88]
If an invalid promql query is sent to trickster it obfuscates the error. For example, if I send `sum(some_metric[1m])` (an invalid query) to prometheus I get a 400 with an error message

```
$ curl -v 'localhost:9090/api/v1/query_range?query=sum(some_metric%5B1m%5D)&start=1533681619.18&end=1533685219.18&step=14&_=1533685048390'l
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 9090 (#0)
> GET /api/v1/query_range?query=sum(some_metric%5B1m%5D)&start=1533681619.18&end=1533685219.18&step=14&_=1533685048390l HTTP/1.1
> Host: localhost:9090
> User-Agent: curl/7.58.0
> Accept: */*
> 
< HTTP/1.1 400 Bad Request
< Access-Control-Allow-Headers: Accept, Authorization, Content-Type, Origin
< Access-Control-Allow-Methods: GET, OPTIONS
< Access-Control-Allow-Origin: *
< Access-Control-Expose-Headers: Date
< Content-Type: application/json
< Date: Tue, 07 Aug 2018 23:41:08 GMT
< Content-Length: 148
< 
* Connection #0 to host localhost left intact
{"status":"error","errorType":"bad_data","error":"parse error at char 21: expected type instant vector in aggregation expression, got range vector"}
```

If I send the same thing to trickster, I get a 502 with no helpful error message.

```
$ curl -v 'localhost:19090/api/v1/query_range?query=sum(some_metric%5B1m%5D)&start=1533681619.18&end=1533685219.18&step=14&_=1533685048390'l
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 19090 (#0)
> GET /api/v1/query_range?query=sum(some_metric%5B1m%5D)&start=1533681619.18&end=1533685219.18&step=14&_=1533685048390l HTTP/1.1
> Host: localhost:19090
> User-Agent: curl/7.58.0
> Accept: */*
> 
< HTTP/1.1 502 Bad Gateway
< Date: Tue, 07 Aug 2018 23:42:02 GMT
< Content-Length: 0
< 
* Connection #0 to host localhost left intact
```

Ideally these sorts of errors would be propagated up, instead of swallowed

**Comment by jranson** [comment]
@jacksontj Thanks for reporting this Thomas! Taking a look now!

**Comment by jranson** [comment]
Fixed in #97
